pages/lobby_page.py

```diff
@@ -21,7 +21,7 @@ def open_rooms_list(self):
             if i.text == 'Rooms':
                 i.click()
 
-    def open_room_by_name(self,name):
+    def open_room_by_name(self, name):
         for i in self.context.driver.find_elements_by_css_selector('.hc-lobby-list-names span.groupchat'):
```

Comment:
Please provide more informative in for cycle at "i" variable. And in line 21-22 please.